src/v/cluster/cluster_recovery_table.cc

```diff
@@ -63,18 +66,25 @@ std::error_code cluster_recovery_table::apply(
     vlog(
       clusterlog.info,
       "Initializing cluster recovery at offset {} with manifest {} from bucket "
-      "{}",
+      "{}, waiting for nodes: {}",
       offset,
-      cmd.value.state.manifest,
-      cmd.value.state.bucket);
+      manifest,
+      bucket,
+      wait_for_nodes);
     _states.emplace_back(
-      std::move(cmd.value.state.manifest),
-      std::move(cmd.value.state.bucket),
-      wait_for_nodes::no);
```

Comment:
had this been hard coded as `no` because code to support `yes` just didn't exist before?

Reply:
That's right, it was just to preserve existing behavior mid-PR.